src/lib/cli_lib/flag.ml

```diff
@@ -262,19 +262,26 @@ module Log = struct
     flag "log-json" no_arg
       ~doc:"Print log output as JSON (default: plain text)"
 
+  let all_levels () =
```

Comment:
Does this need to be a thunk?

`level` and `file_log_level` are evaluated a top-level, so the thunk would be called twice.